### PR TITLE
AttributeContainer extensions

### PR DESCRIPTION
**Pull request by RomanPodymov**
Hello.
Thank you for SwiftyAttributes.
`AttributeContainer` extensions.

### CODE REVIEW

**Review by eddiekaiger — APPROVED** [review]

**Comment by RomanPodymov** [comment]
Hello @eddiekaiger 
I will add more `AttributeContainer` extensions later.

File: SwiftyAttributes/Sources/common/AttributeContainer+SwiftyAttributes.swift [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+//  Created by Roman Podymov on 29/04/23.
+//  Copyright © 2023 Roman Podymov. All rights reserved.
+//
+
+import Foundation
+
+#if canImport(UIKit)
+@available(iOS 15.0, tvOS 15.0, watchOS 8.0, *)
+public extension AttributeContainer {
+    mutating func set(attribute: Attribute) {
```
Comment (eddiekaiger) [inline SwiftyAttributes/Sources/common/AttributeContainer+SwiftyAttributes.swift:14]: not necessary in this PR, but would be neat to also add a custom subscript to `set` and `get` the attributes

**Review by eddiekaiger — APPROVED** [review]

File: SwiftyAttributes/Sources/common/AttributeContainer+SwiftyAttributes.swift (hunk 1 above)
Comment (RomanPodymov) [reply SwiftyAttributes/Sources/common/AttributeContainer+SwiftyAttributes.swift:14]: Hello @eddiekaiger 
Do you mean something like
```
public extension AttributeContainer {
    subscript(attribute: Attribute) -> Any? {
        get { /* */ }
        set { /* */ }
    }
}
```
?

Comment (eddiekaiger) [reply SwiftyAttributes/Sources/common/AttributeContainer+SwiftyAttributes.swift:14]: yep!

**Pull request merged**